angular-rails-app/app/assets/javascripts/home/home.service.js

```diff
@@ -1,4 +1,4 @@
-var API_URL ='http://localhost:3000'
+var API_URL =''
```

Comment:
Note: I was a bit conflicted about removing this, but defaulting it to an empty string ensured we used a relative URL that would work in all environments.

We should, however, ensure this is declared only __once__ in the code. Having it in multiple places is a surefire way to ensure there's a bug introduced.